src/osgEarthAnnotation/MPAnnotationGroupSG.cpp

```diff
@@ -217,8 +217,19 @@ long MPAnnotationGroupSG::addAnnotation(const Style& style, Geometry *geom, cons
     osg::ref_ptr<const InstanceSymbol> instance = style.get<InstanceSymbol>();
     const IconSymbol* iconSym = instance.valid() ? instance->asIcon() : nullptr;
     const TextSymbol* ts = style.get<TextSymbol>();
-    if (ts && ts->priority().isSet())
-        annoDrawable->setPriority(static_cast<float>(style.getSymbol<TextSymbol>()->priority()->eval()));
+    
+ 
+    double priority =0;
+    if (ts && ts->priority().isSet()){
+        priority =style.getSymbol<TextSymbol>()->priority()->eval();    
+        annoDrawable->setPriority(priority);
+        
+    } 
+    if(ts && ts->textPriority().isSet()){
+        priority+=(style.getSymbol<TextSymbol>()->textPriority()->eval()/finePriorityRange);
+        annoDrawable->setPriority(priority);   
+    }
```

Comment:
**Please respect the geneal code format here**
- make visible the operators "priority = style...", "priority += (style"
- put brackets under the condition and not next to it
- group the end bracket with the block it relates to (remove line 226)

Impossible to understand the difference between TextSymbol::priority() and TextSymbol::textPriority
- replace textPriority by priorityFine everywhere